src/implementation/Server/GRPCServer/GRPCServer.ts

```diff
@@ -17,7 +17,7 @@ export default class GRPCServer implements IServer {
   server: IServerType;
   serverImpl: IServerImplType;
   serverCredentials: grpc.ServerCredentials;
-  serverStartupDelay = 1000; // @todo: use health api https://docs.dapr.io/reference/api/health_api/
+  serverStartupDelay = 1000;
```

Comment:
Any reason to remove the TODO comment? I thought we would eventually move to that.

Reply:
It's not needed anymore as the Health check was implemented correctly. The delay is now more a poll check

Reply:
I renamed it to `daprSidecarPollingDelayMs`